docker/Dockerfile

```
@@ -1,11 +1,11 @@
-FROM docker.io/library/ubuntu:focal
+FROM docker.io/library/ubuntu:jammy
 
 ARG DEBIAN_FRONTEND=noninteractive
 ENV TZ=Etc/UTC
-RUN apt-get update && \
+RUN apt-get update &&
```

```suggestion
RUN apt-get update && \
```

The `\` appears to be needed - otherwise I see a:
```
failed to solve with frontend dockerfile.v0: failed to create LLB definition: dockerfile parse error line 6: unknown instruction: APT-GET
```

---

I will fix this ^^